index.js

```diff
@@ -50,13 +50,13 @@ module.exports = {
         type: 'input',
         name: 'subject',
         message: dedent`
-          Write a short, imperative tense description of the change:
+          Write a short, imperative tense description of the change\n  E.g. Fix format in for loop challenge:
```

Comment:
Can you elaborate on "including the type"? I thought type was asked for [in the lines above](https://github.com/freeCodeCamp/cz-freecodecamp/pull/10/files#diff-168726dbe96b3ce427e7fedce31bb0bcL40), unless I'm misunderstanding your comment.